command/server/server.go

```diff
@@ -182,20 +166,14 @@ func setFlags(cmd *cobra.Command) {
 		&params.rawConfig.TxPool.MaxSlots,
 		maxSlotsFlag,
 		command.DefaultMaxSlots,
-		fmt.Sprintf(
-			"maximum slots in the pool. Default: %d",
-			command.DefaultMaxSlots,
-		),
+		"maximum slots in the pool.",
 	)
 
 	cmd.Flags().Uint64Var(
 		&params.rawConfig.BlockTime,
 		blockTimeFlag,
 		defaultConfig.BlockTime,
-		fmt.Sprintf(
-			"minimum block time in seconds. Default: %ds",
-			defaultConfig.BlockTime,
-		),
+		"minimum block time in seconds.",
```

Comment:
Please remove punctuation at the end of flag descriptions, for all commands changed in this PR 🙏

Reply:
Fixed in 4b6856b31184d7b76196a05689f226ecda3a3b3b